tasks/section2.yml

```diff
@@ -396,13 +422,13 @@
       - patch
       - rule_2.2.7
 
-- name: "SCORED | 2.2.7 | PATCH | Ensure NFS and RPC are not enabled"
+- name: "SCORED | 2.2.7 | PATCH | Ensure RPC is not enabled"
   service:
       name: rpcbind
       state: stopped
       enabled: no
   when:
-      - rhel7cis_nfs_rpc_server == false and rpcbind_service_status.stdout == "loaded"
```

Comment:
What is the reason the conditional for `rpcbind_service_status.stdout == "loaded"` was removed from this task?